resources/ts/Components/Integrations/Detail/OrganizersDatalist.tsx

```diff
@@ -0,0 +1,152 @@
+import { Input } from "../../Input";
+import type { Props as FormElementProps } from "../../FormElement";
+import { FormElement } from "../../FormElement";
+import React, { useRef, useState } from "react";
+import type { UiTPASOrganizer } from "../../../types/UiTPASOrganizer";
+import { debounce } from "lodash";
+import { useTranslation } from "react-i18next";
+import { Alert } from "../../Alert";
+import { ButtonIcon } from "../../ButtonIcon";
+import { faTrash } from "@fortawesome/free-solid-svg-icons";
+
+type Props = {
+  onChange: (organizers: UiTPASOrganizer[]) => void;
+  value: UiTPASOrganizer[];
+} & Omit<FormElementProps, "onChange" | "component">;
+
+export const OrganizersDatalist = ({ onChange, value, ...props }: Props) => {
+  const { t } = useTranslation();
+  const [isSearchListVisible, setIsSearchListVisible] = useState(false);
+  const [organizerList, setOrganizerList] = useState<UiTPASOrganizer[]>([]);
+  const [organizerError, setOrganizerError] = useState(false);
+  const organizersInputRef = useRef<HTMLInputElement>(null);
+
+  const handleGetOrganizers = debounce(
+    async (e: React.ChangeEvent<HTMLInputElement>) => {
+      const response = await fetch(`/organizers?name=${e.target.value}`);
+      try {
+        const data = await response.json();
+        if ("exception" in data) {
```

Comment:
```suggestion
        if (!!data && typeof data === "object" && "exception" in data) {
```

Will throw an error on null otherwise

You should probably handle `if !data` seperatly